lighthouse-cli/test/fixtures/dobetterweb/promise_polyfill.js

```diff
@@ -0,0 +1,257 @@
+/*
+ * @license
```

Comment:
ya lets put in third_party. the static server can just look for a request to promise_polyfill and reach across to adjust the absoluteFilePath